lib/routes.py

```diff
@@ -50,16 +50,43 @@ def get_groups():
 @socketio.on('join_group')
 def group_join(data):
   if find_group(data['access_code']):
-    room = data['access_code']
+    group = data['access_code']
+    room = None
```

Comment:
Good idea to change this variable to group instead of room. Even though we are reusing the socket room function it is a good way to differentiate what each variable is for.